### ISSUE TITLE
Private playlists not found by youtube-dl

### ISSUE THREAD

**Issue by eRJe79** [rg3/youtube-dl#8380]
When I tell youtube-dl to download the root of my youtube account (all playlists), it will not find the private playlists and thus they are being skipped. Even though I did add --username and --password in the command.

There are no errors, just the message after execution that only 6 playlists are found (the only public ones).

When I point one (1) private playlist (and the account credentials), the download starts but stops during the first song. For some reason the playlist name is not recognized by youtube-dl and the song is downloaded to a NA folder (not available?).

```
[debug] User config: []
[debug] Command-line args: [u'-v', u'--ignore-config', u'--ignore-errors', u'--no-continue', u'--no-overwrites', u'--yes-playlist', u'--username', u'PRIVATE', u'--password', u'PRIVATE', u'--download-archive', u'/mnt/auvi/youtube/playlist-index.txt', u'--output', u'/mnt/auvi/youtube/%(playlist)s/%(title)s-[%(resolution)s].%(ext)s', u'https://www.youtube.com/watch?v=7qD7h4jqLW0']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.01.09
[debug] Python version 2.7.10 - Linux-3.12.1-eRJe-x86_64-AMD_Opteron-tm-_Processor_6134-with-slackware-14.1
[debug] exe versions: ffmpeg 2.6.3, ffprobe 2.6.3
[debug] Proxy map: {}
[youtube] Downloading login page
[youtube] Logging in
[youtube] 7qD7h4jqLW0: Downloading webpage
[youtube] 7qD7h4jqLW0: Downloading video info webpage
[youtube] 7qD7h4jqLW0: Extracting video information
[youtube] 7qD7h4jqLW0: Downloading DASH manifest
[youtube] 7qD7h4jqLW0: Downloading DASH manifest
WARNING: Requested formats are incompatible for merge and will be merged into mkv.
[debug] Invoking downloader on u'https://r15---sn-5hne6ned.googlevideo.com/videoplayback?id=eea0fb8788ea2d6d&itag=243&source=youtube&requiressl=yes&mn=sn-5hne6ned&nh=IgpwcjA0LmFtczE1Kg4yMTMuNTEuMTU2LjIxMw&mm=31&pl=14&mv=m&ms=au&ratebypass=yes&mime=video/webm&gir=yes&clen=10094664&lmt=1392097755177285&dur=329.000&sver=3&key=dg_yt0&upn=aFkS-6YAnh8&signature=17C491B7AE4FE6EEE9A68CF33763134992A966C4.16D3D105027B891D0BF02D269E0C9686473422D5&mt=1454206935&fexp=9406989,9416126,9420452,9422596,9423661,9423662&ip=94.211.211.188&ipbits=0&expire=1454228633&sparams=ip,ipbits,expire,id,itag,source,requiressl,mn,nh,mm,pl,mv,ms,ratebypass,mime,gir,clen,lmt,dur'
[download] Destination: /mnt/auvi/youtube/NA/Herman van Veen - Laat me-[480x360].f243.webm
[download] 100% of 9.63MiB in 00:00
[debug] Invoking downloader on u'https://r15---sn-5hne6ned.googlevideo.com/videoplayback?id=eea0fb8788ea2d6d&itag=141&source=youtube&requiressl=yes&mn=sn-5hne6ned&nh=IgpwcjA0LmFtczE1Kg4yMTMuNTEuMTU2LjIxMw&mm=31&pl=14&mv=m&ms=au&ratebypass=yes&mime=audio/mp4&gir=yes&clen=10492886&lmt=1427894773895735&dur=329.096&sver=3&key=dg_yt0&upn=aFkS-6YAnh8&signature=8AD0BFE06ACB84ED061D2C697B2FDD61D33072F0.69B0EE178FE2D605051E79A065608611158DAB34&mt=1454206935&fexp=9406989,9416126,9420452,9422596,9423661,9423662&ip=94.211.211.188&ipbits=0&expire=1454228633&sparams=ip,ipbits,expire,id,itag,source,requiressl,mn,nh,mm,pl,mv,ms,ratebypass,mime,gir,clen,lmt,dur'
[download] Destination: /mnt/auvi/youtube/NA/Herman van Veen - Laat me-[NA].f141.m4a
[download] 100% of 10.01MiB in 00:00
[ffmpeg] Merging formats into "/mnt/auvi/youtube/NA/Herman van Veen - Laat me-[480x360].mkv"
[debug] ffmpeg command line: ffmpeg -y -i 'file:/mnt/auvi/youtube/NA/Herman van Veen - Laat me-[480x360].f243.webm' -i 'file:/mnt/auvi/youtube/NA/Herman van Veen - Laat me-[NA].f141.m4a' -c copy -map 0:v:0 -map 1:a:0 'file:/mnt/auvi/youtube/NA/Herman van Veen - Laat me-[480x360].temp.mkv'
Deleting original file /mnt/auvi/youtube/NA/Herman van Veen - Laat me-[480x360].f243.webm (pass -k to keep)
Deleting original file /mnt/auvi/youtube/NA/Herman van Veen - Laat me-[NA].f141.m4a (pass -k to keep)
```

**Comment by dstftw** [comment]
According to the log you pass video URL not a playlist URL. It's impossible to resolve playlist by video URL that's why you get NA for playlist name.

**Comment by eRJe79** [comment]
In the example above, I used the following link in the execution of youtube-dl

https://www.youtube.com/playlist?list=PLFy7RKxDx5bwAVQoFVI3iseqjVMzxXqoq

In the first issue I describe I used

 https://www.youtube.com/channel/UCQ1Wtj_zkcFsA-qge-wG-Eg/playlists 

to point youtube-dl  to the root of my playlists. I've made them all public now, so youtube-dl was able to download almost every song, accept for the restricted ones which is normal behaviour of course.

**Comment by dstftw** [comment]
Fixed in youtube-dl 2016.01.31.

**Comment by eRJe79** [comment]
Thanks!
